Amplify/Core/Support/AmplifyAsyncThrowingSequence.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import Combine
```

Comment:
is this needed?

Reply:
Good catch, this is not needed. Will remove it.